torch/_refs/nn/functional/__init__.py

```diff
@@ -183,6 +224,27 @@ def mish(a: TensorLikeType, inplace: bool = False) -> TensorLikeType:
     return a * torch.tanh(torch.nn.functional.softplus(a))
 
 
+@elementwise_type_promotion_wrapper(
+    type_promoting_args=("a",),
+    type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.DEFAULT,
+)
+def rrelu(
+    a: TensorLikeType,
+    lower: float = 1.0 / 8.0,
+    upper: float = 1.0 / 3.0,
+    inplace: bool = False,
+) -> TensorLikeType:
+    """
+    Reference implementation of torch.nn.functional.rrelu
+    """
+
+    if inplace:
+        raise NotImplementedError
+
+    rhs = refs.uniform(a.shape, low=lower, high=upper, dtype=a.dtype, device=a.device)
```

Comment:
Doesn't this need to be multiplied by `a`? 

https://pytorch.org/docs/master/generated/torch.nn.RReLU.html#torch.nn.RReLU

Reply:
Ugh.  Math fixed.